### PR TITLE
First complete version of the TVMaze api web app

### PR DESCRIPTION
**Pull request by limavfabio**
# Shows page ready for implementation into main!

### This PR adds a full-featured implementation of a web app that displays shows based on the TVMaze API. 
- The user has the possibility of seeing details about the shows.
- Adding comments.
-  Liking his favorite shows.
-  The web app fetches all the information, both the details of the shows, the comments and likes from 2 different external APIs.
- Making all changes persistent across different machines.
- Modal menu working.
- Home page ready
- Paginator displaying an smaller number of pages.
- **Jest** tests over the Comments counter and movies Counter.
- Creation of mocks files to emulate boths APIs

Mi video-11

For documentation purposes, we recorded a video describing the main features of the web app:
https://drive.google.com/file/d/18b0dJoqNA9baCVFtqizlmJhUqD9ytaiT/view?usp=sharing

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,67 @@
...
+   * we have a footer too so I dont want to open the modal that
+   * if the user clicks either on an icon or the footer with the class movie__footer
+   */
+  if (
+    /starBtn-\d+/.test(event.target.id)
+    || /likeBtn-\d+/.test(event.target.id)
+    || event.target.classList.contains('movie__footer')
+  ) {
+    return;
+  }
```
Comment (Lordkaito) [inline src/index.js:45]: - [ ] It looks like the comment popup is not working right now. After I click any movie to try to open it, the app does nothing and the console throws an error saying `event.path is undefined` in this line. So please make sure to work on this issue.

File: src/modules/involvementApi.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const BASE_URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+const GAME_ID = '3bifdQ3qgzMtAvx1V3Pc';
+
+const getLikes = async () => {
+  const result = await fetch(`${BASE_URL}${GAME_ID}/likes`);
+  const data = await result.json();
+  console.log(data);
```
Comment (Lordkaito) [inline src/modules/involvementApi.js:7]: - [ ] Please make sure to remove all console.logs from your codebase to keep your information safe.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @LuisDavidRodriguez, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (LuisDavidRodriguez) [reply src/index.js:45]: Omg!!! it is real?? wow let me check what is going on!

Comment (LuisDavidRodriguez) [reply src/index.js:45]: @Lordkaito can you help me? I'm checking the deployment of the page, I'm not getting that error, can we have a call?

File: src/modules/involvementApi.js (hunk 2 above)
Comment (LuisDavidRodriguez) [reply src/modules/involvementApi.js]: It is done thank you!

**Comment by LuisDavidRodriguez** [comment]
# Important
> Hi there reviewer unfortunately  @Lordkaito does not answer to, get more information about the error he is getting, we check and recheck everything and we can not find nor reproduce the error, I sent the link to another partner and he is telling me that everything is working normally

File: src/index.js (hunk 1 above)
Comment (DeliceLydia) [reply src/index.js:45]: Its working on my side

**Review by DeliceLydia — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### You Rock!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**